### ISSUE TITLE
Dimension page always fail to load

### ISSUE THREAD

**Issue by Matteo-04** [misode/misode.github.io#651]
The dimension page always fails. 
There is the loading screen of gui until the site overloads, in any version even without selecting any preset, it just can't be used. 

Please fix it, thanks

**Comment by klinbee** [comment]
Yeah due to the large size of the overworld preset. Was going to make an Issue myself.

**Comment by misode** [comment]
This is also an issue in the vscode extension, and thus not trivial to fix. See https://github.com/SpyglassMC/Spyglass/issues/1241

**Comment by misode** [comment]
It does actually load for me, but it takes a long time

**Comment by Matteo-04** [comment]
it has never given me much loading problems, it's just few days that Chrome crash on opening the page. 

I'm not sure cause I haven't used it in the past few weeks but I noticed that it doesn't work since the gui update (selection of blocks and fields in some editors)

**Comment by Sniper1-1** [comment]
I would like to add that for me the loading of the overworld dimension is much worse now than before, but while I do receive multiple "this page stopped responding" type messages I can wait it out. However, despite this eventually after it did load and I started opening up the different collapsed entries the page fully crashed with an "out of memory" type error.
